tests/react-test-app/package.json

```diff
@@ -6,8 +6,9 @@
     "start": "react-native start"
   },
   "dependencies": {
-    "react": "15.3.2",
-    "react-native": "^0.37.0",
+    "invariant": "^2.2.2",
```

Comment:
Sure, I can make it a dev dependency. It doesn't make much of a difference unless someone wants to deploy the test app somewhere though.